js/error.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+(function () {
+  var template = document.querySelector('#error')
+    .content
+    .querySelector('div');
+
+  window.error = {
+    handler: function (errorMessage) {
+      var element = template.cloneNode(true);
+      element.querySelector('.error__message').textContent = errorMessage;
+      document.querySelector('main').appendChild(element);
+
+      document.addEventListener('keydown', errorKeydownHandler);
```

Comment:
```suggestion
      document.addEventListener('keydown', documentKeydownHandler);
```

https://up.htmlacademy.ru/javascript/18/criteries#d4